src/app/containers/Article/index.test.jsx

```diff
@@ -1,39 +1,40 @@
 import React from 'react';
 import { shouldShallowMatchSnapshot } from '../../helpers/tests/testHelpers';
 import ArticleContainer from './index';
-import { articleDataNews, articleDataPersian } from './fixtureData';
+import { articleDataNews } from './fixtureData';
```

Comment:
Also see the MetaData point in https://github.com/bbc/simorgh/pull/1244#discussion_r254813161

Reply:
In theory, the body of the article would be tested in both English & Persian in the ArticleMain snapshots -- this test now just checks if ArticleMain is called as expected.

I hadn't noticed the metadata issue -- that somewhat undermines the idea above. I'll have to look into it.

Reply:
Hm, I think Metadata only showed up in ArticleContainer because it's a shallow snapshot. 😕 Maybe I do the same for ArticleMain, so long as it doesn't feel like we're losing anything from the deep render.

Reply:
Switched back to shallow rendering, which ensures Metadata container is called. Don't know how to address the ServiceContextProvider point -- as I said above, I'm not sure this was being tested before.

Reply:
In this container we're implementing the change between different service data, so it would be useful to still have a test for persian as well as News. e.g. the single `200 status code should render correctly` could become `200 status code should render News article correctly` and can add a `200 status code should render Persian article correctly`. 

You should be able to have this work by changing the object `goodData` below to be called `newsData200response` and also adding a `persianData200response` which can be passed into the ArticleContainer in a unit test.

Reply:
Following @bcmn's comment at the bottom of the PR and since the tests have been moved to the ArticleMain, I'm happy with addressing testing context throughout the application more holistically in another PR, rather than in this error status codes PR.

Reply:
Agreed with this, can we have an issue opened please